#ifndef EXPRESSION_VALS_HPP
#define EXPRESSION_VALS_HPP

#include <string>


namespace ExpressionValues
{

class ExpressionNeutral{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};


class ExpressionHappy{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionSad{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionSurprised{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionAngry{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionEvil{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionShy{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

class ExpressionCunning{

public:
    static const std::string LEFT_EYEBROW;
    static const std::string RIGHT_EYEBROW;
    static const std::string MOUTH;
    static const std::string EYELIDS;
};

}

#endif // EXPRESSION_VALS_HPP
